src/eos-metrics-instrumentation.c

```diff
@@ -103,7 +202,7 @@ record_startup (GDBusProxy *dbus_proxy,
           {
             g_warning ("Error unsubscribing from systemd signals: %s.",
                        error->message);
-            g_error_free (error);
+            g_clear_error (&error);
```

Comment:
nit: Recommend undoing this change or going whole hog on the replacement of `g_error_free` with `g_clear_error`.